src/Config.php

```diff
@@ -1175,7 +1175,6 @@ protected function getDefaults()
                 ],
             ],
             'liveeditor'                  => true,
-            'canonical'                   => !empty($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : '',
```

Comment:
Mind if we leave this key and set value to `null`? That's what I have it as in 3.3. Just so we can keep track of keys in one spot.